<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Neutron Dosimetry Trace Recognizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionRecent_Projects"/>
    <addaction name="actionRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Guide"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOriginal"/>
    <addaction name="actionGrayscale"/>
    <addaction name="actionBlackWhite"/>
    <addaction name="actionBlack_Background"/>
    <addaction name="actionWhite_Background"/>
    <addaction name="separator"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionContour"/>
    <addaction name="separator"/>
    <addaction name="actionFill"/>
    <addaction name="actionBorder"/>
    <addaction name="actionCenter"/>
    <addaction name="separator"/>
    <addaction name="actionFit_to_Window"/>
    <addaction name="actionNormal_Size"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_Image"/>
    <addaction name="actionSave_Image"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionProjects"/>
    <addaction name="actionOptions"/>
    <addaction name="actionStatistics"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOriginal"/>
   <addaction name="actionGrayscale"/>
   <addaction name="actionBlackWhite"/>
   <addaction name="actionBlack_Background"/>
   <addaction name="actionWhite_Background"/>
   <addaction name="separator"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionContour"/>
   <addaction name="separator"/>
   <addaction name="actionFill"/>
   <addaction name="actionBorder"/>
   <addaction name="actionCenter"/>
   <addaction name="separator"/>
   <addaction name="actionNormal_Size"/>
   <addaction name="actionFit_to_Window"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
margin: 0px;
border: 0px;</string>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Projects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="treeView">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
margin: 0px;
border: 0px;</string>
   </property>
   <property name="windowTitle">
    <string>Statistics</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
margin: 0px;
border: 0px;</string>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <property name="styleSheet">
     <string notr="true">margin: 0px;
background-color: rgb(232, 232, 232);
border: 0px;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <attribute name="title">
         <string>Image Preprocessor</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>341</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);</string>
         </property>
         <property name="title">
          <string>Threshold</string>
         </property>
         <widget class="QSlider" name="horizontalSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>311</width>
            <height>19</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>161</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Automatic OTSU Threshold</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Trace Resolver</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Colors</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/new_project.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/open_project.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/open_file.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionRecent_Files">
   <property name="text">
    <string>Recent Files</string>
   </property>
  </action>
  <action name="actionRecent_Projects">
   <property name="text">
    <string>Recent Projects</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/close_project.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/exit.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOriginal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <disabledon>:/ico/ico/image_original.ico</disabledon>
    </iconset>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
   <property name="toolTip">
    <string>Original Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/image_grayscale.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Grayscale</string>
   </property>
   <property name="toolTip">
    <string>Grayscale Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionBlackWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/image_blackwhite.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Black-White</string>
   </property>
   <property name="toolTip">
    <string>Black-White Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionUser_Guide">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/user_guide.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/help.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionProjects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Projects</string>
   </property>
   <property name="toolTip">
    <string>Projects</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/undo.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/redo.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy_Image">
   <property name="text">
    <string>Copy Image</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/close_project.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/save.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/save_all.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionBorder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/edge.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Border</string>
   </property>
   <property name="toolTip">
    <string>Trace border</string>
   </property>
  </action>
  <action name="actionContour">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/contour.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Contour</string>
   </property>
   <property name="toolTip">
    <string>Trace Contour</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/center.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
   <property name="toolTip">
    <string>Trace Center</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/zoom_in.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/zoom_out.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionFit_to_Window">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/zoom_fit.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Fit to Window</string>
   </property>
  </action>
  <action name="actionNormal_Size">
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/zoom_100.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
  </action>
  <action name="actionBlack_Background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/black.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Black Background</string>
   </property>
  </action>
  <action name="actionWhite_Background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/white.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>White Background</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/ellipse.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/ico/ico/fill.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
